Description: Fix errors with automake 1.11.2: `pkglibdir' is not a legitimate directory for `DATA'
Author: Gregorio Guidi <gregorio.guidi@gmail.com>
Origin: https://bugs.kde.org/show_bug.cgi?id=290719#c3

diff -Nur valgrind-3.7.0.orig/coregrind/Makefile.am valgrind-3.7.0/coregrind/Makefile.am
--- valgrind-3.7.0.orig/coregrind/Makefile.am
+++ valgrind-3.7.0/coregrind/Makefile.am
@@ -495,7 +495,8 @@
 	m_gdbserver/powerpc-altivec64l.xml
 
 # so as to make sure these get copied into the install tree
-pkglib_DATA  = $(GDBSERVER_XML_FILES)
+vgdatadir = $(pkglibdir)
+vgdata_DATA  = $(GDBSERVER_XML_FILES)
 
 # so as to make sure these get copied into the tarball
 EXTRA_DIST  += $(GDBSERVER_XML_FILES)
diff -Nur valgrind-3.7.0.orig/Makefile.am valgrind-3.7.0/Makefile.am
--- valgrind-3.7.0.orig/Makefile.am
+++ valgrind-3.7.0/Makefile.am
@@ -61,7 +61,8 @@
 # default.supp, as it is built from the base .supp files at compile-time.
 dist_noinst_DATA = $(SUPP_FILES)
 
-pkglib_DATA = default.supp
+vgdatadir = $(pkglibdir)
+vgdata_DATA = default.supp
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = valgrind.pc
